classes/helper.php

```diff
@@ -69,5 +79,28 @@ public static function reason_display(int $reason): string {
                 return (string) $reason;
         }
     }
-}
 
+    /**
+     * Returns a formatted time duration in m:s.ms format.
+     * @param float $duration
+     * @return string
+     * @throws \Exception
+     */
+    public static function duration_display(float $duration): string {
+        $ms = round($duration * 1000, 0);
```

Comment:
I think there are some edge cases with this not working, I think use sprintf would be easy

![image](https://user-images.githubusercontent.com/187449/146286969-5839fddf-c163-43d9-993e-c3e61dffa26d.png)

Also can you please right align this column